config/locales/en.yml

```
@@ -48,6 +48,19 @@ en:
         secret_answer: 'The answer to your security question'
         continue: 'Continue with saving'
         cancel: 'Cancel saving and resume form'
+      confirm_email:
+        heading: 'Check your email address is correct'
+        description: 'Make sure this is correct before finishing'
+      saved:
+        heading: 'Your form has been saved'
+        success_1: 'We have sent a one-off link to '
+        success_2: ' which you can use to resume this form within 28 days. After that time, your saved information will be deleted.'
```

```suggestion
        success: "We have sent a one-off link to %{email} which you can use to resume this form within 28 days. After that time, your saved information will be deleted."
```
Could we interpolate the email into the locale? Then I _think_ this might work in the template:
`t('presenter.save_and_return.saved.success', email: confirmed_email)`